certora/specs/Reverts.spec

```diff
@@ -102,38 +83,31 @@ rule excessivePreliquidationWithAssetsReverts(env e, address borrower, uint256 s
     requireInvariant preLCFConsistent();
     requireInvariant preLIFConsistent();
 
-    PreLiquidation.Market m = MORPHO.market_(currentContract.ID);
-
     // Ensure that no interest is accumulated.
     // Safe require as the invariant ID == marketParams().id() holds, see ConsistentInstantion hashOfMarketParamsOf.
-    require m.lastUpdate == e.block.timestamp;
-
-    PreLiquidation.Position p = MORPHO.position_(currentContract.ID, borrower);
+    require MORPHO.lastUpdate(currentContract.ID) == e.block.timestamp;
 
-    mathint ltv = getLtv(borrower);
+    uint256 ltv = getLtv(borrower);
 
-    mathint preLIF = computeLinearCombination(ltv,
+    uint256 preLIF = computeLinearCombination(ltv,
                                               currentContract.LLTV,
                                               currentContract.PRE_LLTV,
                                               currentContract.PRE_LIF_1,
-                                              currentContract.PRE_LIF_2) ;
+                                              currentContract.PRE_LIF_2);
 
-    // Safe require as implementation would revert with InconsistentInput.
-    require seizedAssets > 0;
+    uint256 seizedAssetsQuoted = summaryMulDivUp(seizedAssets, mockPrice(), ORACLE_PRICE_SCALE());
```

Comment:
Here you're requiring `uint256` are you not? Also note, since math is being summarized the computation can't revert.

Reply:
I agree that the `require_uint256` are explicit but, the changes you want to merge math can't overflow. I'm under the impression that it's the same, what am I missing?

Reply:
Not sure to understand what you are refering to. Here I just removed the `require_uint256` which was not necessary since `summaryMulDivUp` does (and did) return a uint256 already

Reply:
Ok, that makes sense. This where I was unsure about the unsound `require_uint256`, so no issue. Resolving this thread.